### ISSUE TITLE
String not localized in web footer

### ISSUE THREAD

**Issue by qosobrin** [sympa-community/sympa#257]
The [web footer file](https://github.com/sympa-community/sympa/blob/master/default/web_tt2/footer.tt2) contains a string `Powered by Sympa` that is not localized :

```
<!-- footer.tt2 -->
<footer role="contentinfo">

<img src="[% icons_url %]/favicon_sympa.png" alt="[%|loc%]SYMPA logo[%END%]" width="20" height="20"/> 

<a href="http://www.sympa.org"> Powered by Sympa [% version %]</a>
</footer>
<!-- end footer.tt2 -->
``` 

I would correct it myself in the source, but I do not know the procedure you follow to create new localized strings. Is it enough to change the source? How are these new localized strings reflected into the translation site? 

Regards.

**Comment by ikedas** [comment]
Yes, changing the source is enough. Catalog for strings will be updated sometime at least when new version is released, then changes are influenced to translation server.
